home-manager/install.nix

```diff
@@ -1,4 +1,6 @@
-{ home-manager, runCommand }:
+{ home-manager, runCommand
+, shellHook ? "exec ${home-manager}/bin/home-manager init --switch --no-flake"
```

Comment:
I'm not very comfortable allowing wholesale replacement of the shell hook. Perhaps it would be sufficient to add a parameter `extraSwitchOptions` or similar?